about.html

```diff
@@ -0,0 +1,98 @@
+        <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+                    
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="style_about.css">
+        <title>The Concert's Webpage</title>
+    </head>
+<body>
+<!-- about -->
+<section id="about" class="about">
+    <!-- header -->
+    <header>
+        <nav class="navbar">
+            <a href="index.html"><img class="goat_logo_top" src="images/goat_top.PNG" alt="goat logo"><img></a>
+            <div>
+            <button class="hamburger">
+                <div class="bar"></div>
+            </button>
+        </div>
+            <ul class="menu" >
+                <li><a href="about.html">About</a></li>
+                <li><a href="#welcome">Program</a></li>
+                <li><a href="#welcome">Join</a></li>
+                <li><a href="#welcome">Sponsor</a></li>
+                <li><a href="#welcome">News</a></li>
+                <li><a href="#welcome">CC Campaign</a></li>
+            </ul>
+        </nav>
+            <div class="mobile-nav">
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="index.html#event">Event</a></li>
+                <li><a href="index.html#about">Logo</a></li>
+            </ul>
+            </div>
+    </header>
+
+    <h2 class="hello_about">Welcome you all!</h2>
+    <h1>THE GREATEST OF ALL TIME 2023</h1>
+    <p id="about_description"> Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+        Voluptatem, magni ut sint cumque eius fugit error deserunt necessitatibus 
+        modi tempore repellendus. Et, corrupti voluptatibus quis quo nisi, dolor 
+        minima perferendis neque nemo dolores fugiat harum, provident eum possimus </p>
+    <p id="contact">Please contact us per Email for any further questions at <span>theconcert@theconcert.com</span></p>
+    <div id="logo">
+    <h2>The Greatest 2023 Logo</h2>
+    <div class="line"></div>
+    <p id="logo_copyright">The logo of THE GREATEST 2023 is property and copyrighted by THE GREATEST inc.
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+        Quae incidunt reprehenderit qui dolor tenetur cum repellendus dolores voluptatem perspiciatis explicabo.
+    </p>
+    <div class="long_line"></div>
+    <img class="goat_logo" src="images/goat_top.PNG" alt="goat logo">
+    <div class="long_line"></div>
+    </div>
+        <div id="past_editions">
+            <h2 id="past_title">See the pasts GREATEST CONCERTS</h2>
+            <p id="past_text">Take a look at past editions of the concerts in The Random Kingdom and Random Island</p>
+            <div id="past_photos">
+            <img id="past_pictures" src="images/photo_1.PNG" alt="past event 2013">
+            <img id="past_pictures" src="images/photo_2.PNG" alt="past event 2011">
+            </div>
+        </div>
+    </div>
```

Comment:
- [ ] All page elements have similar spacing, borders, font size, shadows, and paddings than the original design.
- [ ] Kindly take a look at what you need to improve on this section:
  - Font size of titles and paragraphs 👍 
  - Add divider after titles 👍 

> |     | Your design                      | Original design                      |     |
> | --- | ----------------------------------- | ------------------------------------ | --- |
> |     | <img width="770" alt="Screen Shot 2023-02-09 at 18 56 22" src="https://user-images.githubusercontent.com/74501523/217866918-0f35387e-71eb-42ec-97d4-22f59abd20fe.png"> | <img width="770" alt="Screen Shot 2023-02-09 at 18 56 48" src="https://user-images.githubusercontent.com/74501523/217867063-d8169dd7-5083-4b50-900b-ba7e4693fade.png"> |

Reply:
Fixed! ✅ 

![image](https://user-images.githubusercontent.com/108192399/217924650-3157cc1c-a363-431f-a3ad-32985937b6a4.png)

Reply:
There goes the below section , too 🔽 

![image](https://user-images.githubusercontent.com/108192399/217925915-19c0f2c7-1a14-48f8-86a1-d6cd0287e5b1.png)